### PR TITLE
FOGL-7811 fledge docker installation steps added in  quickstart guide

### PR DESCRIPTION
**Pull request by ashish-jabble**

### CODE REVIEW

File: docs/quick_start/installing.rst [hunk 1]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
     }
 
 Now, it's time to restart Fledge. Thereafter you will see Fledge is running with PostgreSQL.
+
+
+Using Docker Containerizer to install Fledge
+#############################################
+
+Fledge is provided a private registry. No auth/TLS yet on this private registry for docker container.
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:206]: Fledge Docker containers are provided in a private repository. This repository has no authentication or encryption associated with it.

File: docs/quick_start/installing.rst [hunk 2]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+Useful instructions are given below for it's setup:
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:208]: The following steps describe how to install Fledge using these containers:

File: docs/quick_start/installing.rst [hunk 3]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+- Edit the daemon.json file, whose default location is /etc/docker/daemon.json on Linux, If the daemon.json file does not exist, create it. Assuming there are no other settings in the file, it should have the following contents:
+
+.. code-block:: console
+
+    { "insecure-registries":["52.3.255.136:5000"] }
+
+- Restart Docker for the changes to take effect. sudo systemctl restart docker.service
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:216]: Use code-block for restart example

File: docs/quick_start/installing.rst [hunk 4]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+- Check using "sudo docker info" command, you should have following in output:
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:218]: Check using the command

.. code-block:: console

     sudo docker info

You should see the following output:

File: docs/quick_start/installing.rst [hunk 5]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+.. code-block:: console
+
+    Insecure Registries:
+    52.3.255.136:5000
+    127.0.0.0/8
+
+You may also refer the docker documentation for the setup `here <https://docs.docker.com/registry/insecure/>`_.
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:226]: You may also refer to the Docker documentation....

File: docs/quick_start/installing.rst [hunk 6]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+Ubuntu 20.04
+~~~~~~~~~~~~
+
+- To pull the docker registry
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:231]: Docker should probably always have an uppercase D as it is a name.

File: docs/quick_start/installing.rst [hunk 7]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+    docker pull 54.204.128.201:5000/fledge:latest-ubuntu2004
+
+- To run the docker container
+
+.. code-block:: console
+
+    docker run -d --name fledge -p 8081:8081 -p 1995:1995 -p 8082:80 54.204.128.201:5000/fledge:latest-ubuntu2004
+
+Here, GUI is forwarded to host port 8082, it can be any port and omitted if port 80 is free.
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:243]: The GUI is forwarded to port 8082 on the host machine in the above example, ...

File: docs/quick_start/installing.rst [hunk 8]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+- Now you can see both Fledge and Fledge-GUI is running. You can check below commands in your host machine.
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:245]: It is possible to check if Fledge and the Fledge GUI are running by using the following commands on the host machine

File: docs/quick_start/installing.rst [hunk 9]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+.. code-block:: console
+
+    Fledge: curl -sX GET http://localhost:8081/fledge/ping
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:249]: Separate these out and make then ful commands in both cases.

File: docs/quick_start/installing.rst [hunk 10]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+    GUI: http://localhost:8082
+
+- To attach to running container
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:252]: To attach to the running container

File: docs/quick_start/installing.rst [hunk 11]
```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
...
+
+.. code-block:: console
+
+    docker exec -it fledge bash
+
+.. note::
+    For Ubuntu 18.04 setup, you just need to replace ubuntu2004 with ubuntu1804.
+    At the moment only ubuntu20.04 and ubuntu18.04 (x86_64) images are available.
```
Comment (MarkRiddoch) [inline docs/quick_start/installing.rst:260]: Images are currently only available for Ubuntu version 18.04 and 20.04.

_Are there only Intel based or do we have Arm as well?_

**Review by MarkRiddoch — CHANGES_REQUESTED** [review]

File: docs/quick_start/installing.rst (hunk 11 above)
Comment (ashish-jabble) [reply docs/quick_start/installing.rst:260]: only Intel based

**Review by MarkRiddoch — APPROVED** [review]

**Pull request merged**